telepresence/startup.py

```diff
@@ -24,30 +24,42 @@
 
 from telepresence.runner import Runner
 
+KUBECTL = "kubectl"
+
+OC = "oc"
+
```

Comment:
Could you move these "constants" into the function? They're not used anywhere else, and that's probably for the best.

Thank you for the fix!

Reply:
Fixed in 2dc56cd